client/webserver/site/src/html/wallets.tmpl

```diff
@@ -714,7 +717,25 @@
             <td data-tmpl="voting"></td>
           </tr>
         </tbody>
-      </table>{{/* TODO: ADD MANUAL INPUT */}}
+      </table>
+      <p class="mt-3">
+        <label for="customVspUrl">Or add custom VSP URL:</label>
+      </p>
+      <div class="input-group has-validation">
+        <input 
+          id="customVspUrl"
+          type="url" 
+          class="form-control" 
+          placeholder="https://" 
+          aria-label="Custom VSP URL"
+        />
+        <div class="input-group-append">
+          <button 
+          id="customVspSubmit"
+          class="btn btn-outline-secondary" 
+          type="button">Add</button>
+        </div>
```

Comment:
```suggestion
      <div class="d-flex align-items-stretch w-100">
        <input 
          id="customVspUrl"
          type="url" 
          class="flex-grow-1" 
          placeholder="https://" 
          aria-label="Custom VSP URL"
        >
        <button id="customVspSubmit" class="btn btn-outline-secondary" type="button">Add</button>
```
Please skip the cryptic bootstrap classes and keep keep elements with few attributes on a single line. And HTML5 does not use self-closing tags.

Reply:
Thanks, fixed.